database/query/profiles.sql

```diff
@@ -40,11 +40,11 @@ SELECT * FROM entity_profiles WHERE profile_id = $1 AND entity = $2;
 SELECT * FROM profiles JOIN entity_profiles ON profiles.id = entity_profiles.profile_id
 WHERE profiles.project_id = $1 AND profiles.id = $2;
 
--- name: GetProfileByID :one
-SELECT * FROM profiles WHERE id = $1;
+-- name: GetProfileInfoByID :one
```

Comment:
This PR itself is good, but do you think it makes sense to just add the project id to the other calls in profiles.sql for future when we add more handlers or might refactor them?

Reply:
OK, just added the project ID without changing the name. I'll look into the other SQL calls in another PR